src/Mod/Part/App/FCBRepAlgoAPI_BooleanOperation.cpp

```diff
@@ -109,18 +109,18 @@ void FCBRepAlgoAPI_BooleanOperation::Build() {
         RecursiveAddArguments(myOriginalTools.First());
         if (!myTools.IsEmpty()) {
             myOperation = BOPAlgo_FUSE; // fuse tools together
-            Build();
+            BRepAlgoAPI_BooleanOperation::Build();
```

Comment:
I think the intension was to do a recursion here, that way we can add a `myOperation == BOPAlgo_FUSE` path later, if needed. Infinite recursion won't happen here.
```suggestion
            Build();
```